### DISCUSSION TITLE
Ability to choose to save skins/boot image

### DISCUSSION THREAD

**Discussion by wajdzik** [Skekfly/garlicos-updater#10] [Ideas]
As in title

**Comment by Skekfly** [comment]
Ok I keep the idea

**Reply by Skekfly** [reply]
https://github.com/Skekfly/garlicos-updater/issues/11

**Comment by Skekfly** [comment]
this is done with the last version (v1.3.0).
By default, the conf is kept. There is now a variable to reset confs (please read the README).

**Reply by Skekfly** [reply]
let me know if there is some problem
